webapp/channels/src/components/drafts/utils.ts

```diff
@@ -50,5 +50,12 @@ const errorCodeToErrorMessage = defineMessages<ScheduledPostErrorCode>({
 });
 
 export function getErrorStringFromCode(intl: IntlShape, errorCode: ScheduledPostErrorCode = 'unknown') {
+    const textDefinition = errorCodeToErrorMessage[errorCode];
```

Comment:
Nitpicking, but this could probably be replaced with `const textDefinition = errorCodeToErrorMessage[errorCode] ?? errorCodeToErrorMessage.unknown` to avoid the multiple calls to `formatMessage` and the multiple return statements

Reply:
Done.